src/Text/SlamSearch/Parser/Values.purs

```diff
@@ -0,0 +1,155 @@
+module Text.SlamSearch.Parser.Values where
+
+import Control.Apply
+import Control.Alt
+import Control.Alternative
+import Data.Foldable
+import Data.Array (length)
+import Data.Either
+
+import Text.Parsing.Parser
+import Text.Parsing.Parser.Combinators
+import Text.SlamSearch.Parser.Utils
+import Text.SlamSearch.Parser.Tokens
+
+data Value =
+  Value String
```

Comment:
This constructor is weirdly named because all of these tags are `Value`. Maybe `RawText` or `Text` would be better.